treecorr/catalog.py

```diff
@@ -1118,8 +1118,12 @@ def checkForNaN(self, col, col_str):
         """
         if col is not None and np.any(np.isnan(col)):
             index = np.where(np.isnan(col))[0]
-            self.logger.warning("Warning: NaNs found in %s column.  Skipping rows %s."%(
-                                col_str,str(index.tolist())))
+            if len(index) < 200:
+                self.logger.warning("Warning: %d NaNs found in %s column.  Skipping rows %s."%(
+                                    len(index),col_str,str(index.tolist())))
+            else:
+                self.logger.warning("Warning: %d NaNs found in %s column.  Skipping rows starting %s."%(
+                                    len(index),col_str,str(index[:10].tolist())))
```

Comment:
These two lines need a unit test.  Something semi-gratuitous is fine, but we want the test to touch these lines to make sure all the syntax is (and remains) correct.

Reply:
Done - I've extended one of the tests to make one example where there's > 10 nans (there's already one with a single nan).